### PR TITLE
Add german language

### PR DESCRIPTION
**Pull request by Y0ngg4n**

### CODE REVIEW

File: locale/de-de/movietrailer.intent [hunk 1]
```diff
@@ -0,0 +1,3 @@
+(spiel|spiele|beginn|beginne|start|starte) (trailer|film preview|preview) {{query}}
+(spiel|spiele|beginn|beginne|start|starte) {{query}} (trailer|film preview|preview)
```
Comment (JarbasAl) [inline locale/de-de/movietrailer.intent:2]: are some of these unstranslated or just the same in english?

Comment (Y0ngg4n) [reply locale/de-de/movietrailer.intent:2]: thats ok. we say this in german too. Its "eingedeutscht" in german

Comment (JarbasAl) [reply locale/de-de/movietrailer.intent:2]: then add "eingedeutscht" and any other variations that should also trigger :)

Comment (Y0ngg4n) [reply locale/de-de/movietrailer.intent:2]: No i mean not the word "eingedeutscht", this words are "eingedeutscht". This means that they are so common in german daily language that they where added to the german dictionary ;)

**Pull request merged**